uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -202,6 +208,37 @@ def test_can_enable_false_on_unsupported_kernel_flavor(
         )
         assert (msg, "") == capsys.readouterr()
 
+    @pytest.mark.parametrize(
+        "kernel_version,meets_min_version",
+        (
+            ("3.5.0-00-generic", False),
+            ("4.2.0-00-generic", False),
+            ("4.10.0-00-generic", True),
+            ("5.0.0-00-generic", True),
+        ),
+    )
+    def test_can_enable_false_on_unsupported_min_kernel_version(
+        self, kernel_version, meets_min_version, capsys, entitlement
+    ):
+        """"When on an unsupported kernel version, can_enable returns False."""
+        unsupported_kernel = copy.deepcopy(dict(PLATFORM_INFO_SUPPORTED))
+        unsupported_kernel["kernel"] = kernel_version
+        with mock.patch("uaclient.util.get_platform_info") as m_platform:
+            m_platform.return_value = unsupported_kernel
+            entitlement = LivepatchEntitlement(entitlement.cfg)
+            if meets_min_version:
+                assert entitlement.can_enable()
+            else:
+                assert not entitlement.can_enable()
+        if meets_min_version:
+            msg = ""
+        else:
+            msg = (
+                "Livepatch is not available for kernel {}.\n"
+                "Minimum kernel version required: 4.3\n".format(kernel_version)
```

Comment:
Where does this "4.3" come from? Isn't the 4.4 the cutoff? (Did we ever even release a 4.3 Ubuntu kernel?)

Reply:
I was just using an example here, but probably best to use the expected output that ua-contracts will be sending us and rework comparative examples related to that. Fixing now.